Library/Formula/kafka.rb

```diff
@@ -14,7 +14,7 @@ class Kafka < Formula
 
   depends_on "gradle"
   depends_on "zookeeper"
-  depends_on :java => "1.7+"
+  depends_on :java => "1.7"
```

Comment:
@derekbassett I mean to leave a comment why we use `depends_on :java => "1.7"`.

If you want something like this:
```ruby
depends_on :java => ["1.7", :build]
depends_on :java => ["1.7+", :run]
```
It won't work under current JavaDependency implementation. I can make it work with a PR, but I will leave it to @mikemcquaid to decide whether we should follow this approach.